src/main/scala/overflowdb/codegen/CodeGen.scala

```diff
@@ -621,16 +675,156 @@ def writeConstants(outputDir: JFile): JFile = {
+              case Cardinality.ZeroOrOne =>
+                s"""value match {
+                   |    case null | None => None
+                   |    case someVal:${containedNode.nodeTypeClassName} => Some(someVal)
+                   |  }""".stripMargin
+              case Cardinality.List =>
+                s"""value match {
+                   |    case null | None => Nil
+                   |    case someVal:${containedNode.nodeTypeClassName} => List(someVal)
+                   |    case jCollection: java.lang.Iterable[_] => jCollection.asInstanceOf[java.util.Collection[${containedNode.nodeTypeClassName}]].iterator.asScala.toList
+                   |    case lst: List[_] => value.asInstanceOf[List[${containedNode.nodeTypeClassName}]]
+                   |  }""".stripMargin
+            }
+            s"""  case "${containedNode.localName}" => this._${containedNode.localName} = $s"""
+          }
+          .mkString("\n")
+
+        s"""override def setProperty[A](key:String, value: A):Unit = {
+           |  key match {
+           |$vanillaKeys
+           |$containedKeys
+           |  case _ => PropertyErrorRegister.logPropertyErrorIfFirst(getClass, key)
+           |  }
+           |}""".stripMargin
+      }
+
+      val specificProperty2body: String = {
+        val vanillaKeys = keys
+          .map { key =>
+            Cardinality.fromName(key.cardinality) match {
+              case Cardinality.One | Cardinality.List =>
+                s"""  case "${key.name}" => this._${camelCase(key.name)}"""
+              case Cardinality.ZeroOrOne =>
+                s"""  case "${key.name}" => this._${camelCase(key.name)}.orNull""".stripMargin
+            }
+          }
+          .mkString("\n")
+
+        val containedKeys = nodeType.containedNodesList
+          .map { containedNode =>
+            Cardinality.fromName(containedNode.cardinality) match {
+              case Cardinality.One | Cardinality.List =>
+                s"""  case "${containedNode.localName}" => this._${containedNode.localName}"""
+              case Cardinality.ZeroOrOne =>
+                s"""  case "${containedNode.localName}" => this._${containedNode.localName}.orNull""".stripMargin
+            }
+          }
+          .mkString("\n")
+
+        s"""override def specificProperty2(key:String): AnyRef = {
+           |  key match {
+           |$vanillaKeys
+           |$containedKeys
+           |  case _ => null
+           |  }
+           |}""".stripMargin
+      }
+
+      val specificProperty1body: String = {
+        val vanillaKeys = keys
+          .map { key =>
+            Cardinality.fromName(key.cardinality) match {
+              case Cardinality.One =>
+                s"""  case "${key.name}" => if(this._${camelCase(key.name)} == null) VertexProperty.empty[A]
+                   |      else new OdbNodeProperty(-1, this, key, this._${camelCase(
+                  key.name
```

Comment:
more spurious newlines here - maybe an editor setting?

Reply:
Indeed -- I made the mistake of attempting to use scalafmt, which also caused the mess in the initial commit. Thanks for spotting that!